### PR TITLE
Make minor grammar corrections/updates to async/callbacks

### PR DESCRIPTION
**Pull request by jchue**
The first of the split out and revised changes originally in #1612

### CODE REVIEW

File: 1-js/11-async/01-callbacks/article.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ For instance, we can schedule such actions using `setTimeout`.
 
 There are other real-world examples of asynchronous actions, e.g. loading scripts and modules (we'll cover them in later chapters).
 
-Take a look at the function `loadScript(src)`, that loads a script with the given `src`:
+Take a look at the function `loadScript(src)` that loads a script with the given `src`:
```
Comment (paroche) [inline 1-js/11-async/01-callbacks/article.md]: I preferred it with the comma.

File: 1-js/11-async/01-callbacks/article.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ We can use this function like this:
 loadScript('/my/script.js');
 ```
 
-The script is executed "asynchronously", as it starts loading now, but runs later, when the function has already finished.
+The script is executed "asynchronously", as it starts loading now but runs later when the function has already finished.
 
 If there's any code below `loadScript(…)`, it doesn't wait until the script loading finishes.
```
Comment (paroche) [inline 1-js/11-async/01-callbacks/article.md]: I prefer keeping at least the first comma. Second more optional, but I still think it works better with it than without it. Ideas settle in my head better w/ the commas.

File: 1-js/11-async/01-callbacks/article.md [hunk 3]
```diff
@@ -109,13 +109,13 @@ loadScript('https://cdnjs.cloudflare.com/ajax/libs/lodash.js/3.2.0/lodash.js', s
...
 
-Here we did it in `loadScript`, but of course, it's a general approach.
+Here we did it in `loadScript`, but of course it's a general approach.
 
 ## Callback in callback
 
 How can we load two scripts sequentially: the first one, and then the second one after it?
 
-The natural solution would be to put the second `loadScript` call inside the callback, like this:
+The natural solution would be to put the second `loadScript` call inside the callback like this:
```
Comment (paroche) [inline 1-js/11-async/01-callbacks/article.md]: I guess I like commas more than the committer. I would leave it.

File: 1-js/11-async/01-callbacks/article.md [hunk 4]
```diff
@@ -151,11 +151,11 @@ loadScript('/my/script.js', function(script) {
 });
 ```
 
-So, every new action is inside a callback. That's fine for few actions, but not good for many, so we'll see other variants soon.
+So every new action is inside a callback. That's fine for a few actions but not good for many, so we'll see other variants soon.
```
Comment (paroche) [inline 1-js/11-async/01-callbacks/article.md]: Again, I prefer it as was, w/ comma.

File: 1-js/11-async/01-callbacks/article.md [hunk 5]
```diff
@@ -303,7 +303,7 @@ See? It does the same, and there's no deep nesting now because we made every act
 
 It works, but the code looks like a torn apart spreadsheet. It's difficult to read, and you probably noticed that one needs to eye-jump between pieces while reading it. That's inconvenient, especially if the reader is not familiar with the code and doesn't know where to eye-jump.
 
-Also, the functions named `step*` are all of single use, they are created only to avoid the "pyramid of doom." No one is going to reuse them outside of the action chain. So there's a bit of namespace cluttering here.
+Also, the functions named `step*` are all of single use — they are created only to avoid the "pyramid of doom". No one is going to reuse them outside of the action chain. So there's a bit of namespace cluttering here.
```
Comment (paroche) [inline 1-js/11-async/01-callbacks/article.md]: It's true, we're putting periods outside of quotes, British-style. And dash is a bit better than comma.

**Comment by iliakan** [comment]
@paroche not merging it yet, waiting for changes, right?

**Comment by paroche** [comment]
Well, I was making my opinions known. For the 4 where I commented on the commas, I would not merge them on my own, since I don't prefer them. The last one, line 306, w/ the "pyramid of doom", I would merge on its own, but don't recall seeing that as an option.

**Comment by iliakan** [comment]
@jchue, could you update your PR, so that @paroche approves and merges it?

**Comment by jchue** [comment]
I do not think commas should be used to indicate pauses in speech, nevertheless I have made the updates as requested.

**Comment by paroche** [comment]
Thanks.

**Pull request closed without merging**